firmware/boot/bootloader.c

```diff
@@ -88,12 +89,12 @@ _Noreturn static void modifyStackPointerAndStartApp(const uint32_t *address)
     // program counter accordingly.
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Warray-bounds"
-    uint32_t app_sp    = address[0];
-    uint32_t app_start = address[1];
+    uint32_t boot_sp    = address[0];
+    uint32_t boot_start = address[1];
```

Comment:
I believe I accidently changed these to boot. Could you change these back to app instead of boot